### ISSUE TITLE
Use as a business partnership or friendship plugin?

### ISSUE THREAD

**Issue by Momshroom** [GeorgH93/MarriageMaster#183]
### Help request

**Problem**

The wiki says that you can change this into a friendship plugin with no code changes, but doesn't say how.   I can see changing the language file and then aliasing all the commands as an option.  Is that what you meant?
If not and there's a feature to change the base command from marry to something else, I'm missing it and would greatly appreciate instructions.   
If there isn't a feature and my first idea was correct, would you like me to PR a language file for others to use who might also want to use it as a business partnership or friendship plugin? 
If you really wanted I could attempt to PR a customizable command name.

**What I have tried**

Reading the code and the wiki and changing the language file.

**Comment by GeorgH93** [comment]
There is no feature, the idea is to change the language file and config options.
You can change all commands in the language file (the first one in the list is the one that will be displayed in the help).
It looks like the top level command uses the same command list as the `/marry marry` command, I will fix that on the weekend.

If you create a language file and share it that would be really great.

**Comment by GeorgH93** [comment]
[This build](https://ci.pcgamingfreaks.at/job/MarriageMaster/260/artifact/artifacts/MarriageMaster-2.2.14-Release.jar) should allow you to change the main and marry command independently.

**Comment by Momshroom** [comment]
It doesn't seem to be working for me.  I'm not sure if I'm doing something wrong, but I reverted to the default language file and changed ONLY the main command and it still doesn't recognize my new command and does recognize /marry.
I confirmed it is using my language file by changing the language and author fields and it shows those loading.

I'm changing it on line 324:
  Main:
    - partner
    
Thanks for any assistance
